com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/DepthOfFieldCoCReproject.compute

```diff
@@ -54,10 +54,10 @@ void KMain(uint3 dispatchThreadId : SV_DispatchThreadID)
     float coc3 = cocBR.x;
     float coc4 = cocBR.z;
 #else
-    float coc1 = LOAD_TEXTURE2D_X(_InputCoCTexture, posInputs.positionSS - uint2(1u, 0u)).x; // Left
-    float coc2 = LOAD_TEXTURE2D_X(_InputCoCTexture, posInputs.positionSS - uint2(0u, 1u)).x; // Top
-    float coc3 = LOAD_TEXTURE2D_X(_InputCoCTexture, posInputs.positionSS + uint2(0u, 1u)).x; // Bottom
-    float coc4 = LOAD_TEXTURE2D_X(_InputCoCTexture, posInputs.positionSS + uint2(1u, 0u)).x; // Right
+    float coc1 = LOAD_TEXTURE2D_X(_InputCoCTexture, ClampAndScaleUVPostProcessTextureForPoint(posInputs.positionSS) - uint2(1u, 0u)).x; // Left
```

Comment:
Id say cache this result in a register. I dont trust dxc being smart enough to catch the renundant code below.

Reply:
Yeah, good point, thanks!